provider/cmd/pulumi-resource-xyz/provider.ts

```diff
@@ -18,7 +18,7 @@ import * as provider from "@pulumi/pulumi/provider";
 import { StaticPage, StaticPageArgs } from "./staticPage";
 
 export class Provider implements provider.Provider {
-    constructor(readonly version: string) { }
+    constructor(readonly version: string, readonly schema: string) { }
```

Comment:
Totally curious - mostly for my own education - how does this work?  We don't seem to be passing `schema` to anything - what is it that ensures this causes `GetSchema` to return the schema?